silverback/cluster/settings.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+import tomlkit
+from pydantic import BaseModel, Field, ValidationError, model_validator
+from typing_extensions import Self
+
+PROFILE_PATH = Path.home() / ".silverback" / "profile.toml"
+DEFAULT_PROFILE = "default"
+
+
+class AuthenticationConfig(BaseModel):
+    """Authentication host configuration information (~/.silverback/profile.toml)"""
+
+    host: str = "https://account.apeworx.io"
+    client_id: str = Field(default="lcylrp34lnggGO-E-KKlMJgvAI4Q2Jhf6U2G6CB5uMg", alias="client-id")
```

Comment:
question: I'm sure it's not, but just want to be sure. This isn't secret, correct?

Reply:
This is public OAuth client ID (same as putting it in JS client-side), we need this value for the CLI to act as a client to our auth system, because all clients have to be registered

I think I need to do something separate w/ PKCE, but that might also only be a browser thing

Reply:
Actually Fief automatically does PKCE with the CLI public client:
https://docs.fief.dev/going-further/pkce/#use-pkce-with-fief
https://github.com/fief-dev/fief-python/blob/05a17b6d94cefd33ce0aeb060a1fd5f91c1b1ebc/fief_client/integrations/cli.py#L225